core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -348,6 +353,10 @@ oppia.directive('conversationSkin', [function() {
           }, 100);
         };
 
+        $scope.iframeReloadExprolation = function() {
```

Comment:
Spelling is wrong. Also, start functions with a verb: $scope.reloadExploration()